packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -375,6 +376,7 @@ export function TransferPanelMain({
   } = useBalance({ provider: l2.provider, walletAddress })
 
   const isSwitchingL2Chain = useIsSwitchingL2Chain()
+  const destinationAddressDenylist = useDestinationAddressDenylist()
```

Comment:
Hmm, what if instead of a hook we just exposed a utility function like `isOnDenylist` or something. Seems cleaner than having to pass a prop :thinking: